services/red/tests/artiso.AdsdHotel.Red.Tests/artiso.AdsdHotel.Red.Tests.csproj

```diff
@@ -1,19 +1,25 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>net5.0</TargetFramework>
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
Use `net5.0`, or even better remove the `TargetFramework` tag from the projects since it's defined globally in the [Directory.Build.props](https://github.com/artiso-solutions/adsd-hotel/blob/main/services/Directory.Build.props).

Reply:
I wonder why that works, `netcoreapp5.0` doesn't exist as a TFM oO